test/run-cdn.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This needs `chmod +x`!

Reply:
ad4c130cb50a2c862ce91c81d77fccffd0d9da11 done :confetti_ball: